lib/challenge.js

```diff
@@ -60,6 +92,19 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+var Pet = function pet(name, species, breed, noise) = {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+}
+
+var Pet = new getPetInfo();
+
+Pet.prototype.makeNoise = function makeNoise() {
+      return this.noise;
+  };
+};
```

Comment:
This prototype method would be great but for some syntactic problems. There is only one opening curly brace to match here, but there are two closing curly braces.

Overall, this answer gets enough right, but suffers from some serious errors. Here is a fixed version to compare and contrast in case that helps: